### PR TITLE
Docker service test

### PR DESCRIPTION
**Pull request by abbgrade**
Hello,

this is a very simple test, that tests if the docker service is running, that explains why the other tests are failing with timeout exceptions.

Thanks,
Steffen

### CODE REVIEW

File: test/Docker.DotNet.Tests/ISystemOperations.Tests.cs [hunk 1]
```diff
@@ -15,6 +17,15 @@ public ISystemOperationsTests()
             _client = new DockerClientConfiguration(new Uri("npipe://./pipe/docker_engine")).CreateClient();
         }
 
+        [Fact]
+        public void DockerService_IsRunning()
+        {
+            var services = ServiceController.GetServices();
```
Comment (jterry75) [inline test/Docker.DotNet.Tests/ISystemOperations.Tests.cs]: Only one question. Why did you remove the:
```C#
using (var services = ...)
{
  //...
}
```
`Component` is disposable so shouldn't we do that?

**Comment by abbgrade** [comment]
You are right. Its better to use `using` here.

**Comment by jterry75** [comment]
Thank you for the contribution!
